src/libstd/rt/util.rs

```diff
@@ -144,56 +144,6 @@ pub fn abort(args: fmt::Arguments) -> ! {
     let _ = write!(&mut w, "{}", args);
     let msg = str::from_utf8(&w.buf[0..w.pos]).unwrap_or("aborted");
     let msg = if msg.is_empty() {"aborted"} else {msg};
```

Comment:
@eddyb It seems to cover real cases: the first if the buffer is not utf8 (though with fmt changing to require utf8 maybe that's impossible), and the second if the abor t message is empty. Neither of those seem to be related to the easter egg.